### ISSUE TITLE
Error with IPInfo.io (:ipinfo_io): "Please specify a valid lookup"

### ISSUE THREAD

**Issue by arnlen** [alexreisner/geocoder#1008]
Here is the error with version `1.3.1` of the Geocoder gem:

```
Geocoder::ConfigurationError: Please specify a valid lookup for Geocoder (:ipinfo_io is not one of: :dstk, :esri, :google, :google_premier, :google_places_details, :yahoo, :bing, :geocoder_ca, :geocoder_us, :yandex, :nominatim, :mapbox, :mapquest, :opencagedata, :ovi, :here, :baidu, :geocodio, :smarty_streets, :okf, :postcode_anywhere_uk, :geoportail_lu, :test, :baidu_ip, :freegeoip, :geoip2, :maxmind, :maxmind_local, :telize, :pointpin, :maxmind_geoip2).
```

Configuration file:

```rb
# config/initializers/geocoder.rb
Geocoder.configure(

  # IP address geocoding service (see https://github.com/alexreisner/geocoder#ip-address-services for supported options):
  :ip_lookup => :ipinfo_io,

  # geocoding service request timeout, in seconds (default 3):
  :timeout => 5,
)
```

**Comment by alexreisner** [comment]
Sorry, that lookup is not included in the current gem version. Will be available in 1.3.2, and you can use it now by installing the gem from the current HEAD.

**Comment by oshields** [comment]
@arnlen This was released and seems to work now if you bundle update geocoder.  Note that the keys of the response are different from freegeoip.  country_code is not country, etc.
